### PR TITLE
Update community supported README: legal disclaimer

### PR DESCRIPTION
**Pull request by cwarnermm**
Added legal disclaimer to README, as requested by Product.

### CODE REVIEW

**Comment by cwarnermm** [comment]
@larkox - Please confirm whether or not you'd prefer to remain listed as a maintainer of this repository. Happy to update this PR as needed.

**Review by larkox — APPROVED** [review]

**Review by hanzei — CHANGES_REQUESTED** [review]
Blocking until the discussion in https://community.mattermost.com/private-core/pl/51hbdxfn7tbbuboys4siw8peea has been resolved

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
+# :warning: Disclaimer
+
+**This repository is community supported and not maintained by Mattermost. Mattermost disclaims any and all liability for integrations, including Third Party Integrations and Mattermost Integrations. All integrations are provided "AS IS", and may be used at your own risk. Integrations may be modified or discontinued at any time.**
```
Comment (cwarnermm) [inline README.md:3]:
```suggestion
# Disclaimer

**This repository is community supported and not maintained by Mattermost. Mattermost disclaims liability for integrations, including Third Party Integrations and Mattermost Integrations. Integrations may be modified or discontinued at any time.**
```

**Comment by cwarnermm** [comment]
@hanzei - Legal's updated guidance and language for the legal disclaimer for community-supported integrations has been applied to this PR.

**Comment by cwarnermm** [comment]
@hanzei - This PR is ready for your final approval.

**Comment by cwarnermm** [comment]
@hanzei - This PR is ready for your final approval.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
+# Disclaimer
+
+**This repository is community supported and not maintained by Mattermost. Mattermost disclaims liability for integrations, including Third Party Integrations and Mattermost Integrations. Integrations may be modified or discontinued at any time.**
+
 # Mattermost MS Teams Meetings Plugin
 
 [![Build Status](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-msteams-meetings/master)](https://circleci.com/gh/mattermost/mattermost-plugin-msteams-meetings)
 [![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-msteams-meetings/master)](https://codecov.io/gh/mattermost/mattermost-plugin-msteams-meetings)
 [![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-msteams-meetings)](https://github.com/mattermost/mattermost-plugin-msteams-meetings/releases/latest)
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-msteams-meetings/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-msteams-meetings/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
```
Comment (hanzei) [inline README.md:11]:
```suggestion

**Maintainer:** [@mickmister](https://github.com/mickmister)

```

**Review by hanzei — APPROVED** [review]

**Pull request merged**